src/main/sensors/gyro.h

```diff
@@ -29,6 +29,8 @@
 
 #include "pg/pg.h"
 
+#define FILTER_FREQUENCY_MAX 4000 // maximum frequency for filter cutoffs (nyquist limit of 8K max sampling)
```

Comment:
Would be nice to have a `STATIC_ASSERT` to make sure that this isn't higher than the loop nyquist frequency.

Reply:
Not sure how to do this. The actual gyro loop is determined at runtime and even by which gyro. For example the Bosch gyro runs at 3.2KHz so the limit is actually 1600.

Reply:
Sorry, I meant the maximum attainable loop frequency, to avoid getting in the same situation we got with the removal of 32 kHz support.

Reply:
Well that's what adding and using the define for all the limits is doing - centralizing the max possible. Before it was hardcoded values scattered throughout the code. I expect this single define to be the hard limit.

I guess we could add a `#define MAX_GYRO_SAMPLE_HZ 8000` and define the `FILTER_FREQUENCY_MAX` from that - but seems kind of superfluous. Or maybe I'm still not understanding.

Reply:
I thought of doing something like what we do for `samplingTime` in `validateAndFixGyroConfig()` but in static fashion, but looking into how it's calculated it the required complexity seems to be making it not worthwhile.